presto-product-tests/src/main/java/com/facebook/presto/tests/cli/PrestoCliTests.java

```diff
@@ -171,6 +179,11 @@ protected void launchPrestoCliWithServerArgument(String... arguments)
                 prestoClientOptions.add("--krb5-disable-remote-service-hostname-canonicalization");
             }
         }
+
+        if (ldapAuthentication) {
```

Comment:
Irritating as it is to modify tempto, I think we should do that (either to echo the password or export an environment variable when running the CLI) so that we don't go directly against David's suggestion.